x-pack/plugins/maps/public/routes/map_page/map_app/map_app.tsx

```diff
@@ -486,6 +487,18 @@ export class MapApp extends React.Component<Props, State> {
       <TopNavMenu
         setMenuMountPoint={this.props.setHeaderActionMenu}
         appName={APP_ID}
+        badges={
+          this.props.savedMap.isManaged()
+            ? [
+                getManagedContentBadge(
+                  i18n.translate('xpack.maps.mapController.managedMapDescriptionTooltip', {
+                    defaultMessage:
+                      'This map is managed by Elastic. Changes here must be saved to a new map.',
```

Comment:
Nit - how about removing "here" so that "Changes here must be saved to a new map." becomes "Changes must be saved to a new map."